holoviews/plotting/util.py

```diff
@@ -72,6 +73,67 @@ def collate(obj):
         raise Exception(undisplayable_info(obj))
 
 
+def isoverlay_fn(obj):
+    """
+    Determines whether object is a DynamicMap returning (Nd)Overlay types.
+    """
+    return isinstance(obj, DynamicMap) and (isinstance(obj.last, CompositeOverlay))
+
+
+def compute_overlayable_zorders(obj, path=[]):
+    """
+    Traverses the Callables on DynamicMap to determine which objects
```

Comment:
Isn't the traversal of DynamicMap one of the things it needs to do as opposed to the only thing it needs to do?

How about:

> Traverses an overlayable  composite container to determine which objects are associated with specific (Nd)Overlay layers by z-order, taking DynamicMap Callables into account.  Returns a mapping between the zorders of each layer and a corresponding lists of objects.